Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -103,77 +97,19 @@ def load_logs(dir)
     logs
   end
 
-  def create_gist(files, descr)
-    post("/gists", { "public" => true, "files" => files, "description" => descr })["html_url"]
-  end
-
-  def new_issue(repo, title, body, auth)
-    post("/repos/#{repo}/issues", { "title" => title, "body" => body }, auth)["html_url"]
-  end
-
-  def http
-    @http ||= begin
-      uri = URI.parse("https://api.github.com")
-      p = ENV["http_proxy"] ? URI.parse(ENV["http_proxy"]) : nil
-      if p.class == URI::HTTP || p.class == URI::HTTPS
-        @http = Net::HTTP.new(uri.host, uri.port, p.host, p.port, p.user, p.password)
-      else
-        @http = Net::HTTP.new(uri.host, uri.port)
-      end
-      @http.use_ssl = true
-      @http
-    end
+  def create_gist(files, description)
+    data = { "public" => true, "files" => files, "description" => description }
+    GitHub.open("https://api.github.com/gists", data)["html_url"]
   end
```

Comment:
Yeh, perhaps. It feels a little weird if they are only used in this file, though, so I'd rather hold off for now.